### ISSUE TITLE
[Bug][iOS][A11Y][Text input] Text inputs not reachable anymore with external keyboard or switch control

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-flutter#500]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-flutter/issues) for duplicate or closed feature requests

### Your test device

iPhone 14+

### OS Version

iOS26

### App version

0.6.0 (509)

### Describe the issue

Text Input is not reachable anymore with Full Keyboard Access or switch access, the focus goes through the header, the customize sheet and the copy code button but not the text inputs

### Expected Behavior

Text input shall be usable with keyboard or switch control

**Comment by pya35** [comment]
For components implementing text fields (text input, password, pin code), the text field doesn't have any trait. Everything is in the Label property. That may be the root cause of the issue.

<img width="508" height="530" alt="" src="https://github.com/user-attachments/assets/4fdde0d2-f657-4fac-bf35-60e35e2d7a81" />

**Comment by pya35** [comment]
linked to #367
